k-distribution/tests/pyk/imp-import-graph.expected

```diff
@@ -0,0 +1,178 @@
+ARRAYSYNTAX
+ARRAYSYNTAXLIST
+AUTOCASTS
+AUTOFOLLOW
+BASICK
+BASICKMLSYNTAX
+BASICKSORTBOOL
+BOOL
+BOOLBASICK
+BOOLBOOLSYNTAX
+BOOLSYNTAX
+BOOLSYNTAXSORTBOOL
+BUILTINIDTOKENS
+BYTESCONCRETE
+BYTESCONCRETEBYTESHOOKED
+BYTESHOOKED
+BYTESHOOKEDBYTESSYNTAX
+BYTESHOOKEDSTRINGSYNTAX
+BYTESSYNTAX
+BYTESSYNTAXSTRINGSYNTAX
+CONFIGCELLS
+CONFIGCELLSKCELLS
+CONFIGCELLSRULELISTS
+DEFAULTLAYOUT
+DIGRAPH
+DOMAINSSYNTAX
+DOMAINSSYNTAXBOOLSYNTAX
+DOMAINSSYNTAXIDSYNTAX
+DOMAINSSYNTAXSORTK
+DOMAINSSYNTAXSTRINGSYNTAX
+DOMAINSSYNTAXUNSIGNEDINTSYNTAX
+FLOATSYNTAX
+ID
+IDCOMMON
+IDCOMMONIDSYNTAX
+IDCOMMONSTRING
+IDIDCOMMON
+IDSYNTAX
+IDSYNTAXPROGRAMPARSING
+IDSYNTAXPROGRAMPARSINGBUILTINIDTOKENS
+IMP
+IMPBASICK
+IMPBOOL
+IMPIMPSYNTAX
+IMPINT
+IMPKIO
+IMPMAP
+IMPSET
+IMPSYNTAX
+IMPSYNTAXBOOLSYNTAX
+IMPSYNTAXID
+IMPSYNTAXINTSYNTAX
+INT
+INTBOOL
+INTCOMMON
+INTCOMMONBOOL
+INTCOMMONINTSYNTAX
+INTINTCOMMON
+INTINTKORE
+INTKEQUAL
+INTKORE
+INTKOREINTCOMMON
+INTKOREKEQUAL
+INTSYNTAX
+INTSYNTAXUNSIGNEDINTSYNTAX
+K
+KAMBIGUITIES
+KAST
+KASTBASICK
+KASTBUILTINIDTOKENS
+KASTKSTRING
+KASTSORTKBOTT
+KAUTOCASTS
+KAUTOFOLLOW
+KBOTTOMSORT
+KBOTTOMSORTSORTKBOTT
+KCELLS
+KCELLSKAST
+KEQUAL
+KEQUALBOOL
+KEQUALKEQUALKORE
+KEQUALKEQUALSYNTAX
+KEQUALKORE
+KEQUALKOREBOOL
+KEQUALKOREKEQUALSYNTAX
+KEQUALSYNTAX
+KEQUALSYNTAXBASICK
+KEQUALSYNTAXBOOL
+KIO
+KIOINT
+KIOLIST
+KIOSTRING
+KKREWRITE
+KKSEQSYMBOLIC
+KKSORTLATTICE
+KLOCATIONS
+KLOCATIONSINTSYNTAX
+KLOCATIONSSTRINGSYNTAX
+KRECORDPRODUCTIONS
+KREFLECTION
+KREFLECTIONBASICK
+KREFLECTIONSTRING
+KREQUIRESENSURES
+KREWRITE
+KSEQ
+KSEQKAST
+KSEQKTOPSORT
+KSEQSYMBOLIC
+KSEQSYMBOLICKSEQ
+KSEQSYMBOLICKVARIABLESYNTAX
+KSEQSYMBOLICMLSYNTAX
+KSORTLATTICE
+KSORTLATTICEKBOTTOMSORT
+KSORTLATTICEKTOPSORT
+KSORTPREDICATES
+KSTRING
+KTERM
+KTERMAUTOCASTS
+KTERMAUTOFOLLOW
+KTERMKREWRITE
+KTERMKSEQSYMBOLIC
+KTERMKSORTLATTICE
+KTERMRECORDPRODUCTIONS
+KTERMSORTPREDICATES
+KTOPSORT
+KTOPSORTSORTKBOTT
+KVARIABLESYNTAX
+LANGUAGEPARSING
+LANGUAGEPARSINGIDSYNTAXPROGRAMPARSING
+LANGUAGEPARSINGIMP
+LANGUAGEPARSINGIMPSYNTAX
+LANGUAGEPARSINGKTERM
+LIST
+LISTBASICK
+LISTINTSYNTAX
+MAP
+MAPBOOLSYNTAX
+MAPINTSYNTAX
+MAPLIST
+MAPSET
+MINTSYNTAX
+MLSYNTAX
+MLSYNTAXSORTK
+PROGRAMLISTS
+PROGRAMLISTSSORTK
+RECORDPRODUCTIONS
+REQUIRESENSURES
+REQUIRESENSURESBASICK
+RULECELLS
+RULECELLSKCELLS
+RULECELLSRULELISTS
+RULELISTS
+RULETAGSYNTAX
+RULETAGSYNTAXBUILTINIDTOKENS
+SET
+SETBASICK
+SETINTSYNTAX
+SORTBOOL
+SORTK
+SORTKBOTT
+SORTKBOTTSORTK
+SORTPREDICATES
+STDINSTREAM
+STDOUTSTREAM
+STRING
+STRINGCOMMON
+STRINGCOMMONBOOL
+STRINGCOMMONFLOATSYNTAX
+STRINGCOMMONINT
+STRINGCOMMONKEQUAL
+STRINGCOMMONSTRINGSYNTAX
+STRINGKORE
+STRINGKOREKEQUAL
+STRINGKORESTRINGCOMMON
+STRINGSTRINGCOMMON
+STRINGSTRINGKORE
+STRINGSYNTAX
+UNSIGNEDINTSYNTAX
```

Comment:
If we add a new module to domains.md, is this going to fail?
Does `make update-results` work? I see it has no target in the Makefile.

Reply:
Yes, it will fail. `make CHECK=cp` will update the output.

I think we should switch `make update-results` to the `make CHECK=cp` system we use in the various semantics, it's cleaner to me. But that is a big project.

Reply:
What do you mean by `make CHECK=cp` is cleaner?
I find `make update-results` way more intuitive from a naming convention. It's a make target as well.
If you want us to change it, then you should make an issue. Until then I think it's better if we stick to the same standard we use everywhere in our tests.

I removed automerge until we clarify this.